app/routes/index.js

```diff
@@ -2,31 +2,28 @@ import Ember from 'ember';
 import config from 'ember-get-config';
 import ResetScrollMixin from '../mixins/reset-scroll';
 
+const getTotalPayload = '{"size": 0, "from": 0,"query": {"bool": {"must": {"query_string": {"query": "*"}}, "filter": [{"term": {"type.raw": "preprint"}}]}}}';
+
 export default Ember.Route.extend(ResetScrollMixin, {
     model() {
-        var getTotalPayload = '{"size": 0, "from": 0,"query": {"bool": {"must": {"query_string": {"query": "*"}}, "filter": [{"term": {"type.raw": "preprint"}}]}}}';
-
-        var sharePreprintsTotal = Ember.$.ajax({
-                type: 'POST',
-                url: config.SHARE.searchUrl,
-                data: getTotalPayload,
-                contentType: 'application/json',
-                crossDomain: true,
-            }).then(function (results) {
-                return results.hits.total.toLocaleString();
-            });
+        return this.store.query('taxonomy', { filter: { parents: 'null' }, page: { size: 20 } });
+    },
+    setupController(controller, model) {
+        this._super(controller, model);
+        controller.set('currentDate', new Date());
 
-        return Ember.RSVP.hash({
-            theDate: new Date(),
-            subjects: this.store.query('taxonomy', { filter: { parents: 'null' }, page: { size: 20 } }),
-            sharePreprintsTotal: sharePreprintsTotal
-        });
+        // Fetch total number of preprints. Allow elasticsearch failure to pass silently.
+        Ember.$.ajax({
```

Comment:
Also, there's a general convention in ember for data fetching to be owned by the route. (esp wrt the data required for first page load). Even though we don't want the model hook to block on a failed request, it makes sense to keep the basic data in the same area instead of creating a separate file.

Also, since controllers are singletons, we want to rerun the network request on every visit and not simply serve back old state from a previous in-app visit to the page. The general recommendation is to do such cleanup in `setupController`, while crossing your fingers and waiting patiently for routable components...!

It's a good question; this is one of those cases where Ember is... fun.